packages/netlify-cms-core/src/backend.js

```diff
@@ -41,6 +41,7 @@ class LocalStorageAuthStore {
 function prepareSlug(slug) {
   return (
     slug
+      .trim()
```

Comment:
Preview link paths have their individual segments passed through this function, but slug templates go through this function whole, so it seems possible that this could result in untrimmed segments within the slug. Perhaps removing all whitespace from the slug after processing as a precaution could work?

Reply:
@erquhart  did a little bit of testing and I think we don't really need to `trim` each segment of the slug here. Turns out that the  urlHelper `sanitizeURI` turns  all white space to the config's `sanitize_replacement` option and then the `(double|trailing|leading)Replacement` does the final work, thoughts.

Reply:
Alright, sounds like we're covered - thanks for digging into that 👍